input/examples/practitioner-chau-fryer.xml

```diff
@@ -1,9 +1,34 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!-- Compliant with HI Vendor Test Data -->
```

Comment:
Any instance where 'Compliant with HI Vendor Test Data' will need to be explained. Not all elements are in the Vendor Test Data. If you want to keep this assertion in this tranche of work then a blurb will need to be crafted on which parts are compliant with. I would recommend that this goes into a GitHub backlog.

I've logged this once, but it applies to the other examples with this comment (except for Location that cannot assert compliance to HI Vendor Test Data)

Reply:
Removed from examples & added a GitHub backlog item.